RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -652,7 +652,7 @@ bool DAClusterizerInZ_vect::purge(vertex_t& y, track_t& tks, double& rho0, const
   double sumpmin = nt;
   unsigned int k0 = nv;
 
-  std::vector<double> inverse_zsums(nt), arg_cache(nt), eik_cache(nt), pcut_cache(nt);
+  std::vector<double> inverse_zsums(nt), arg_cache(nt), eik_cache(nv), pcut_cache(nt);
```

Comment:
Wrong variable?  eik_cache is indexed by nt, it's pcut_cache that's indexed by nv:

https://github.com/cms-sw/cmssw/blob/bf9dafbcea4d0a9a12bea2e8347ff9594a2d11e9/RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc#L668-L671

Reply:
Brain fog. Thanks.